### DISCUSSION TITLE
Declarative workflow without logging in?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [ariga/atlas#3224] [Q&A]
Hey! I really like atlas project, and I need it currently while developing to apply my schema file and that's it. But it's gotten annoying to have to login so often. Is this part of the paid features?

`atlas schema apply --env local --auto-approve`

The `--env local` is refering to the `atlas.hcl` env section. I don't think I use any premium features. Is there any way to use the declarative workflow without logging in?

**Comment by a8m** [comment]
Hey @Nikola-Milovic,

The default flow does not require login. Can you show how your config file looks like?
